xbmc/pvr/PVRManager.cpp

```diff
@@ -391,6 +392,18 @@ void CPVRManager::ResetProperties(void)
 
 void CPVRManager::Init()
 {
+  std::set<std::string> settingSet;
+  settingSet.insert(CSettings::SETTING_PVRMANAGER_CHANNELMANAGER);
+  settingSet.insert(CSettings::SETTING_PVRMANAGER_GROUPMANAGER);
+  settingSet.insert(CSettings::SETTING_PVRMANAGER_CHANNELSCAN);
+  settingSet.insert(CSettings::SETTING_PVRMANAGER_RESETDB);
+  settingSet.insert(CSettings::SETTING_PVRCLIENT_MENUHOOK);
+  settingSet.insert(CSettings::SETTING_PVRMENU_SEARCHICONS);
+  settingSet.insert(CSettings::SETTING_EPG_RESETEPG);
+  settingSet.insert(CSettings::SETTING_EPG_DAYSTODISPLAY);
+  settingSet.insert(CSettings::SETTING_PVRPARENTAL_ENABLED);
+  CSettings::GetInstance().RegisterCallback(this, settingSet);
```

Comment:
BTW with C++11 and initializer lists this could be written as
```cpp
CSettings::GetInstance().RegisterCallback(this, { CSettings::SETTING_PVRMANAGER_CHANNELMANAGER, ..., CSettings::SETTING_PVRPARENTAL_ENABLED });
```
but that's just cosmetics.